tasks.c

```diff
@@ -1013,105 +1013,99 @@ static void prvAddNewTaskToReadyList( TCB_t * pxNewTCB ) PRIVILEGED_FUNCTION;
             /* There are configNUMBER_OF_CORES Idle tasks created when scheduler started.
              * The scheduler should be able to select a task to run when uxCurrentPriority
              * is tskIDLE_PRIORITY. */
-            configASSERT( ( uxCurrentPriority > tskIDLE_PRIORITY ) || ( xTaskScheduled == pdTRUE ) );
             uxCurrentPriority--;
```

Comment:
The comment above is no longer relevant?

Reply:
It helps to explain uxCurrentPriority is never decreased to value below tskIDLE_PRIORITY.